backend/src/graphql/resolver/ContributionMessageResolver.ts

```diff
@@ -54,8 +54,7 @@ export class ContributionMessageResolver {
       await queryRunner.commitTransaction()
     } catch (e) {
       await queryRunner.rollbackTransaction()
-      logger.error(`ContributionMessage was not successful: ${e}`)
-      throw new Error(`ContributionMessage was not successful: ${e}`)
+      throw new LogError('ContributionMessage was not sent successfully', e)
```

Comment:
Do we want to give the reason to the frontend as well? See contribution message resolver